config/initializers/audited_hacks.rb

```diff
@@ -78,6 +78,7 @@ module AuditedHacks
 
   module ClassMethods
     def audited(options = {})
+      options[:on] = _filter_touch_callback(options[:on])
```

Comment:
I think that if we interntionally have set `touch` for auditing, then it should stay. Just in case we didn't specify anything, then we need to avoid having it set by default. So my suggestion is to do like below. Or do you think we need to be extra careful to avoid `touch`? Will auditing fail on `touch`? 

```suggestion
      options[:on] = options[:on].presence || %i[create update destroy]
```

Reply:
You're right, `:touch` should be allowed is that what we want. Also it makes the code simpler. I pushed the changes.